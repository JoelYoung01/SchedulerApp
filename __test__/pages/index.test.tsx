import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import EditSchedule from "../../pages/edit-schedule";

test("EditSchedule Renders", () => {
  renderer.create(<EditSchedule />);
});
