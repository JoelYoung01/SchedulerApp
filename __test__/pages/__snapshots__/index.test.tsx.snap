// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditSchedule Renders 1`] = `
[
  <div
    className="MuiGrid-root MuiGrid-container printed css-17qsxia-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-dqfxkm-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignLeft css-16taog-MuiTypography-root"
          >
            10.23
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignRight css-1js305m-MuiTypography-root"
          >
            Sun
          </h6>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-dqfxkm-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignLeft css-16taog-MuiTypography-root"
          >
            10.23
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignRight css-1js305m-MuiTypography-root"
          >
            Mon
          </h6>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-dqfxkm-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignLeft css-16taog-MuiTypography-root"
          >
            10.23
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignRight css-1js305m-MuiTypography-root"
          >
            Tue
          </h6>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-dqfxkm-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignLeft css-16taog-MuiTypography-root"
          >
            10.23
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignRight css-1js305m-MuiTypography-root"
          >
            Wed
          </h6>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-dqfxkm-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignLeft css-16taog-MuiTypography-root"
          >
            10.23
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignRight css-1js305m-MuiTypography-root"
          >
            Thu
          </h6>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-dqfxkm-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignLeft css-16taog-MuiTypography-root"
          >
            10.23
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignRight css-1js305m-MuiTypography-root"
          >
            Fri
          </h6>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-dqfxkm-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignLeft css-16taog-MuiTypography-root"
          >
            10.23
          </h6>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1osj8n2-MuiGrid-root"
        >
          <h6
            className="MuiTypography-root MuiTypography-h6 MuiTypography-alignRight css-1js305m-MuiTypography-root"
          >
            Sat
          </h6>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1idn90j-MuiGrid-root"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1i1uysr-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Drew
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-98geom-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Fred
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-7z4rst-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                John
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-12pc05e-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Spencer
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1idn90j-MuiGrid-root"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1cypjok-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Mike
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1i1uysr-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Drew
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-98geom-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Fred
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-7z4rst-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                John
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1idn90j-MuiGrid-root"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-12pc05e-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Spencer
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1cypjok-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Mike
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1i1uysr-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Drew
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-98geom-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Fred
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1idn90j-MuiGrid-root"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-7z4rst-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                John
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-12pc05e-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Spencer
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1cypjok-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Mike
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1i1uysr-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Drew
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1idn90j-MuiGrid-root"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-98geom-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Fred
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-7z4rst-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                John
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-12pc05e-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Spencer
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1cypjok-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Mike
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1idn90j-MuiGrid-root"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1i1uysr-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Drew
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-98geom-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Fred
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-7z4rst-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                John
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-12pc05e-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Spencer
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1i5d9ma-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 css-1idn90j-MuiGrid-root"
        >
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1cypjok-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Mike
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1i1uysr-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Drew
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-98geom-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Fred
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-7z4rst-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                John
              </p>
            </div>
          </div>
          <div
            className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-zzb0zw-MuiPaper-root-MuiCard-root"
          >
            <div
              className="MuiCardContent-root css-1i89k6q-MuiCardContent-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-ety8s4-MuiTypography-root"
              >
                9:30am
                 - 
                10:30am
              </p>
              <p
                className="MuiTypography-root MuiTypography-body1 css-1ptsv0t-MuiTypography-root"
              >
                Drew Accola
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="css-e53awj-MuiStack-root"
  >
    <div>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        Export
      </button>
    </div>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium css-zcbmsk-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      Add Employee
    </button>
  </div>,
]
`;
